code/modules/events/shuttle_catastrophe.dm

```diff
@@ -0,0 +1,31 @@
+/datum/round_event_control/shuttle_catastrophe
+	name = "Shuttle Catastrophe"
+	typepath = /datum/round_event/shuttle_catastrophe
+	weight = 10
+	max_occurrences = 1
+
+/datum/round_event/shuttle_catastrophe
+	var/datum/map_template/shuttle/new_shuttle
+
+/datum/round_event/shuttle_catastrophe/announce(fake)
+	var/cause = pick("was attacked by [syndicate_name()] Operatives", "mysteriously teleported away", "had it's refuelling crew mutiny",
+		"was found with it's engines stolen", "\[REDACTED\]", "flew into the sunset, and melted", "learned something from a very wise cow, and left on it's own",
+		"\[REDACTED\]", "had cloning devices on it")
+
+	priority_announce("Your emergency shuttle [cause]. Your replacement shuttle will be [new_shuttle.name] until further notice.", "CentCom Spacecraft Engineering")
+
+/datum/round_event/shuttle_catastrophe/setup()
+	var/list/valid_shuttle_templates = list()
+	for(var/shuttle_id in SSmapping.shuttle_templates)
+		var/datum/map_template/shuttle/template = SSmapping.shuttle_templates[shuttle_id]
+		if(template.can_be_bought && template.credit_cost < INFINITY) //if we could get it from the communications console, it's cool for us to get it here
```

Comment:
Are you sure you don't want to check `prerequisites_met()`? I'm cool with it (since it means a lot of shuttles that literally never get bought would be shown), but want to make sure this isn't an oversight.

Reply:
Also not being able to get the Daniel from this event is heartbreaking.

Reply:
Yeah, I purposefully left it out. If the crew could IN THEORY get it, it shouldn't be too much trouble to have it.